### ISSUE TITLE
Placeholder pages for modules and clients

### ISSUE THREAD

**Issue by zuiderkwast** [valkey-io/valkey-io.github.io#177]
**Is your feature request related to a problem? Please describe.**

Our docs markdown pages (topics and commands) contain links to `../clients/` and `../modules/` for the lists of such resources. Currently they are broken or redirected to some other page.

**Describe the solution you'd like**

Let's add placeholder pages with some "under construction" or "coming soon" text. It's not ideal, but IMO it's better than incorrect or broken links.

Page URLs that would match the links in the current docs markdown would be

    https://valkey.io/clients/
    https://valkey.io/modules/

#166 will add the clients page but it may take some time before it's ready.

**Comment by stockholmux** [comment]
I agree with not having 404s but at the same time "under construction" pages are not ideal (despite having some.. I'd rather not add more).

Would it make sense to have a minimal interim page or forward? 
`/modules/` forward to `valkey.io/topics/modules-intro`
`/clients/` forward to `valkey.io/topics/protocol`

**Comment by zuiderkwast** [comment]
I'd rather have a "Coming soon" stub page than a redirect to some different content which is not what the visitor was looking for. Ideally, it the stub page shouldn't be completely empty. For clients, we could link to our few clients github repos, e.g.?

> Clients
> -------
>
> Coming soon. In the meantime, find out more about our official clients under each client's repository.
>
> * valkey-py
> * valkey-glide
> * valkey-go
> * iovalkey
> * valkey-java
> * libvalkey

And for modules, I don't have a good suggestion...
